byterun/Makefile.nt

```diff
@@ -59,7 +61,10 @@ libcamlrund.$(A): $(DOBJS)
 	echo " caml/freelist.h caml/minor_gc.h caml/osdeps.h caml/signals.h"\
 	  >> .depend.win32
 	cat .depend >> .depend.win32
-	sed -e '/\.d\.o/q' -e 's/^\(.*\)\.o:/\1.$$(O) \1.$$(DBGO):/' \
+	sed -ne '/\.pic\.o/q' \
+	    -e 's/^\(.*\)\.d\.o:/\1.$$(DBGO):/' \
+	    -e 's/^\(.*\)\.o:/\1.$$(O):/' \
```

Comment:
This never worked - interp.o has additional dependencies when compiled for debug mode.